src/Engine/ProtoCore/Properties/Resources.en-US.resx

```diff
@@ -885,7 +885,7 @@
   </data>
   <data name="ArgumentNullException" xml:space="preserve">
     <value>Value cannot be null.
-Parameter name: {0}. href=ArgumentNullException.html.</value>
+Parameter name: {0}. href=ArgumentNullException.html</value>
```

Comment:
this was preventing these pages from ever loading.

Reply:
Good catch!